libkbfs/block_types.go

```diff
@@ -190,6 +225,27 @@ func (fb FileBlock) DeepCopy(codec kbfscodec.Codec) (*FileBlock, error) {
 	return &fileBlockCopy, nil
 }
 
+// UpdateHash updates the hash of this FileBlock
+func (fb *FileBlock) UpdateHash() kbfshash.RawDefaultHash {
```

Comment:
Do we really need this function too?  I'm not sure we need a way to force the update to happen (even though I know the current code does it that way).  Seems like the block contents should never change once either of these functions is called for the first time.